src/index.js

```diff
@@ -329,6 +347,63 @@ export default class AuthnWidget {
     }
   }
 
+  postEmptyAuthentication() {
+    this.store.dispatch('POST_FLOW', 'authenticate', '{}');
+  }
+
+  registerMfaEventHandler() {
+    Array.from(document.querySelectorAll('[data-mfa-selection]'))
+      .forEach(element => element.addEventListener('click', this.handleMfaDeviceSelection));
+  }
+
+  handleMfaDeviceSelection(evt) {
+    evt.preventDefault();
+    let source = evt.currentTarget;
+    if (source) {
+      let deviceId = source.dataset['mfaSelection'];
+      console.log(deviceId)
```

Comment:
Can we safely remove this or make it a little more descriptive? I've been having trouble lately deciphering the console logging in the widget. Might be good to make logs more meaningful going forward

Reply:
oh yes, definitely! I must've missed removing it...